lib/presets/safe.es6

```diff
@@ -27,4 +27,5 @@ export default {
     removeRedundantAttributes: false,
     removeComments: 'safe',
     sortAttributesWithLists: true,
+    minifyUrls: false,
```

Comment:
`minifyUrls` should be considered as dangerous and shouldn't be enabled by default.

The option is not added in the `max` preset either since it requires the user's configuration.